documentation.md

```diff
@@ -12,6 +12,8 @@ navigation:
 <p>Setup instructions, programming guides, and other documentation are available for each stable version of Spark below:</p>
 
 <ul>
+  <li><a href="{{site.baseurl}}/docs/2.1.2/">Spark 2.1.2</a></li>
```

Comment:
Include 2.2.0 while you're at it? not sure why this hasn't been updated.

Reply:
I think the idea is this is the list of releases that is not the latest.
The link to this page under the "Documentation" tab says "Older Versions and Other Resources"

But I guess it could be a bit confusing?